src/collections/componentSet.ts

```diff
@@ -104,17 +117,22 @@ export class ComponentSet extends LazyCollection<MetadataComponent> {
       registry = input.registry ?? registry;
       tree = input.tree ?? tree;
       inclusiveFilter = input.include;
+      fsDeletePaths = input.fsDeletePaths ?? fsDeletePaths;
```

Comment:
could paths meant for deletions be in the `input`? Is that up to consumers to specify `fsDeletePaths`?